Makefile

```diff
@@ -0,0 +1,27 @@
+test_db:
+	createdb -E UTF8 -O ${PG_USER} ${PG_DB}
+	psql -d ${PG_DB} -c "CREATE EXTENSION postgis;"
+	psql -d ${PG_DB} -c "CREATE EXTENSION hstore;"
+	echo "ALTER USER ${PG_USER} WITH PASSWORD '${PG_PASSWORD}';" | psql -d ${PG_DB}
+
+ingest:
+	docker build --file Dockerfile.osm2pgsql --build-arg PBF_URL=${PBF_URL} -t osm2pgsql .
+	docker run --net=host -it --rm osm2pgsql -c "osm2pgsql --slim --hstore \
```

Comment:
We aren't using the Postgres password here, so Postgres would have to be set up to trust this user.  This line could be changed to `docker run --net=host -it --rm osm2pgsql -c "PGPASSWORD=\"osmpassword\" osm2pgsql --slim --hstore \`